examples/jsm/nodes/accessors/StorageBufferNode.js

```diff
@@ -13,6 +13,8 @@ class StorageBufferNode extends BufferNode {
 
 		this.isStorageBufferNode = true;
 
+    this.readOnly = false;
```

Comment:
Please make sure to adhere the project's code style. This was already mentioned in https://github.com/mrdoob/three.js/pull/28379#issuecomment-2114245807.